src/email_requests/tests.py

```diff
@@ -108,6 +109,62 @@ def test_list_requests(self):
         self.assertEqual(self.request1_json, resp_request1)
         self.assertEqual(self.request1_json, resp_request1)
 
+    def test_mentee_mentor_requests(self):
+        
+        self.request1 = factories.RequestFactory(mentor=self.mentor)
+        self.request2 = factories.RequestFactory(mentee=self.profile)
+        self.request3 = factories.RequestFactory(mentor=self.mentor)
+        self.request4 = factories.RequestFactory(mentee=self.profile)
+
+        self.request1_json = RequestSerializer(self.request1).data
+        self.request2_json = RequestSerializer(self.request2).data
+        self.request3_json = RequestSerializer(self.request3).data
+        self.request4_json = RequestSerializer(self.request4).data
+
+        resp = self.client.get(
+            self.get_url,
+        )
+
+        self.assertEqual(resp.data['count'], 4)
+        self.assertEqual(len(resp.data['results']), 4)
+
+        #order is changed because response is ordered in reverse
+        resp_request1 = resp.data['results'][3]
+        resp_request2 = resp.data['results'][2]
+        resp_request3 = resp.data['results'][1]
+        resp_request4 = resp.data['results'][0]
+
+     
+        self.assertEqual(self.request1_json, resp_request1)
+        self.assertEqual(self.request2_json, resp_request2)
+        self.assertEqual(self.request3_json, resp_request3)
+        self.assertEqual(self.request4_json, resp_request4)
+
```

Comment:
is there one endpoint for both the requests you made and the requests that are made to you?  I think we can make a more usable API than that

Reply:
Are you saying make two different endpoints? Seems redundant if the only use is to display all requests that one is involved in. Ram said this would be best

Reply:
yes, we should do two endpoints in my opinion. we would never show requests you made and requests you've received in the same context.